backend/process-document/src/main/kotlin/com/ritense/processdocument/resolver/DocumentTableValueResolver.kt

```diff
@@ -97,6 +104,22 @@ class DocumentTableValueResolver(
         }
     }
 
+    private fun handleValues(documentId: Document.Id, values: Map<String, Any?>) {
```

Comment:
It looks like more is included in your PR: Case tags and assigneeId. Could you include this in the release notes?